manim/animation/animation.py

```diff
@@ -4,37 +4,39 @@
 __all__ = ["Animation", "Wait"]
 
 
+import typing
 from copy import deepcopy
 
 import numpy as np
 
+if typing.TYPE_CHECKING:
+    from manim.scene.scene import Scene
+
 from .. import logger
 from ..mobject.mobject import Mobject
-
 from ..utils.rate_functions import smooth
 
-
-DEFAULT_ANIMATION_RUN_TIME = 1.0
-DEFAULT_ANIMATION_LAG_RATIO = 0
+DEFAULT_ANIMATION_RUN_TIME: float = 1.0
+DEFAULT_ANIMATION_LAG_RATIO: int = 0
```

Comment:
I believe this could in principle be a float, no?

Reply:
yeah, it has the word ratio so float.